### DISCUSSION TITLE
How can I access the globalOptions in a component?

### DISCUSSION THREAD

**Discussion by iconnoise** [flyntwp/flynt#572] [Q&A] [answered]
I want to access globalOptions that are created in a tab within a component. Unfortunately, I always get “NULL”. Do you have a sample code for me?

**Comment by steffenbew** [comment, marked as answer]
Hey, retrieving fields from a tab within a component should be straightforward: 
While fields in tabs are visually nested, they're available on the top level in the data structure.

Here's an example:
[BlockWysiwyg/functions.php](https://github.com/flyntwp/flynt/blob/master/Components/BlockWysiwyg/functions.php): Definition of tabs (Content | Options) and fields (contentHtml, options).
[BlockWysiwyg/index.twig](https://github.com/flyntwp/flynt/blob/master/Components/BlockWysiwyg/index.twig): Rendering of fields (contentHtml, options.*).

Please also note that `globalOptions` are a different concept to store and retrieve settings for the whole site. With `Options::addGlobal()` you can add options to the Global Options page and retrieve them with `Options::getGlobal()`.

If you require further support, please post your code snippet for further investigation of the problem.
